mixins/itemHelper.ts

```diff
@@ -44,11 +44,9 @@ const itemHelper = Vue.extend({
      * @returns {boolean} Whether the item can be played on this client or not
      */
     canPlay(item: BaseItemDto): boolean {
-      const itemType = item.Type || '';
-      const itemMediaType = item.MediaType || '';
-
       if (
-        [
+        item &&
+        ([
           'Audio',
```

Comment:
Non-blocking, but I preferred the old way of doing this, I think it was much easier to look at